pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/PulsarAdminBuilder.java

```diff
@@ -327,4 +327,11 @@ PulsarAdminBuilder authentication(String authPluginClassName, Map<String, String
      */
     PulsarAdminBuilder setContextClassLoader(ClassLoader clientBuilderClassLoader);
 
+    /**
+     * Allow to control the gzip compression of the HTTP response payload.
+     * By default, "Accept-Encoding: gzip" is added to HTTP requests.
+     *
+     * @param gzipCompressionEnabled whether to enable gzip compression
+     */
+    PulsarAdminBuilder acceptGzipCompression(boolean gzipCompressionEnabled);
```

Comment:
It's sending, not accepting, maybe just `enableGzipCompression`?

Reply:
This is a combined change in this experimental draft PR, sorry for the confusion. 
It is accepting. A client will be sending "Accept-Encoding: gzip" header and the server will decide based on this whether it gzips or not.

Reply:
I understand, but from the normal user, this in effect enables gzip compression, so I prefer to make it express what it is in reality :)

Reply:
I renamed all locations of "gzipCompressionEnabled" with "acceptGzipCompression" so that the expectation is clear. The client will accept gzip compression. I have also updated the javadoc to make it clear.

Reply:
This was moved to #22464